### PR TITLE
Should fix the crusher, again, and the offhand runtime

### LINKED ISSUE #22821: Crushers break hand slots when slotting into back
**Issue:** When the crusher is equipped to the back slot, it leaves a rogue offhand in your non-active hand, which can't be dropped other then by a forcedrop from admins. If this offhand is swapped to, you are then unable to swap hand at all or use that hand for anything.

**Expected Outcome:** The crusher should go into the back slot and the offhand deleted

This is presumably caused by #22712 

@Haevacht 

Before Equipping
After Equipping
Error when trying to swap off the hand

### PR DESCRIPTION
**Pull request by Haevacht**
I hate github most the time.

Fixes #22821, probably. Also fixes the runtime the backup plan caused.

### CODE REVIEW

**Comment by Arianya** [comment]
So what new issue do I get to report tommorow Mr Haevacht?

**Comment by Haevacht** [comment]
Equals signs being unrepentant and becoming sentient to murder everyone.

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -167,6 +169,12 @@
 	..()
 	var/slotbit = slotdefine2slotbit(slot)
 	if(slot_flags & slotbit)
+		var/O = user.is_holding_item_of_type(/obj/item/weapon/twohanded/offhand)
+		if(!O)
```
Comment (optimumtact) [inline code/game/objects/items/weapons/twohanded.dm]: just combine these two lines if(!O || qdeleted(O))

the first condition is always checked first and if true, won't bother trying the second condition and just fall through

Comment (Haevacht) [inline code/game/objects/items/weapons/twohanded.dm]: Neat. Doing that now.

**Pull request merged**